server/expression/comparison.go

```diff
@@ -0,0 +1,41 @@
+package expression
+
+import (
+	"fmt"
+
+	"github.com/kubeshop/tracetest/server/assertions/comparator"
+	"github.com/kubeshop/tracetest/server/expression/value"
+)
+
+func compare(comparatorName string, leftValue, rightValue value.Value) error {
+	comparatorFunction, err := comparator.DefaultRegistry().Get(comparatorName)
+	if err != nil {
+		return fmt.Errorf("comparator not supported: %w", err)
+	}
+
+	if leftValue.IsArray() && comparatorName == "contains" {
+		return compareArrayContains(leftValue, rightValue)
```

Comment:
This way we can use te `contains` operator to check if an array contains a certain value in it. Just like javascript's `includes` function.